### ISSUE TITLE
Wrong example in Firebird 5 Language Reference

### ISSUE THREAD

**Issue by tonne1** [FirebirdSQL/firebird-documentation#184]
Section 8.12.4. RDB$ROLE_IN_USE()

RDB$ROLE_IN_USE Examples
List currently active roles

```
select rdb$role_name
from rdb$database      <--- should be rdb$roles
where rdb$role_in_use(rdb$role_name);
```

**Comment by mrotteveel** [comment]
Thanks for reporting. I fixed this in the Firebird 4.0 and 5.0 Language Reference. I'll release the fixed versions later this week.

**Comment by mrotteveel** [comment]
The fixed versions were published earlier today.
